src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -544,11 +544,16 @@ const CreateImageWizard = () => {
   // This will occur if 'Recreate image' is clicked
   const initialStep = compose?.request ? 'review' : undefined;
 
-  const { isBeta } = useGetEnvironment();
+  const { isBeta, isProd } = useGetEnvironment();
 
   const awsTarget = isBeta() ? awsTargetBeta : awsTargetStable;
   const msAzureTarget = isBeta() ? msAzureTargetBeta : msAzureTargetStable;
-  let initialState = requestToState(composeRequest, distroInfo, isBeta());
+  let initialState = requestToState(
+    composeRequest,
+    distroInfo,
+    isBeta(),
+    isProd()
+  );
```

Comment:
Not very happy with how many arguments this function takes. Might need to do some refactoring in the future. Open to suggestions if anything seems obvious to you, dear reviewer.

Reply:
Can we put the is beta check into our global store?